README.md

```diff
@@ -56,6 +56,7 @@ The following configuration options are available:
 - `DRYDOCK_ENABLE_SENTRY`: Whether to enable sentry. Defaults to `true`.
 - `DRYDOCK_SENTRY_DSN`: The sentry DSN. Defaults to `""`.
 - `DRYDOCK_POD_LIFECYCLE`: Whether to enable pod lifecycle. Defaults to `true`.
+- `NGINX_STATIC_CACHE_CONFIG`: A list of dictionaries with settings for different services to cache their assets in NGINX.
```

Comment:
Could you please include an example of how the NGINX_STATIC_CACHE_CONFIG variable should look like? In this way, the operator knows what keys to add for every service